src/data/meetings.json

```diff
@@ -87,5 +87,17 @@
     "remoteAudio": null,
     "localShare": null,
     "remoteShare": null
+  },
+  "localMedia": {
+    "ID": "localMedia",
+    "title": "Ad Hoc Meeting Space",
+    "startTime": null,
```

Comment:
mind as well remove the times here too. I added a separate commit to my PR to clean up the objects.